core/src/main/scala/kafka/server/DynamicBrokerConfig.scala

```diff
@@ -797,6 +801,9 @@ object DynamicThreadPool {
       case ReplicationConfigs.NUM_REPLICA_FETCHERS_CONFIG => config.numReplicaFetchers
       case ServerLogConfigs.NUM_RECOVERY_THREADS_PER_DATA_DIR_CONFIG => config.numRecoveryThreadsPerDataDir
       case ServerConfigs.BACKGROUND_THREADS_CONFIG => config.backgroundThreads
+      case RemoteLogManagerConfig.REMOTE_LOG_MANAGER_COPIER_THREAD_POOL_SIZE_PROP => config.remoteLogCopierThreads
```

Comment:
Get the copier thread size using the below way, we don't want to expose the remoteLogManager configs in KafkaConfig class:

```suggestion
      case RemoteLogManagerConfig.REMOTE_LOG_MANAGER_COPIER_THREAD_POOL_SIZE_PROP => config.remoteLogManagerConfig.remoteLogManagerCopierThreadPoolSize() 
```

same for other configs